openstack/manifest.json

```diff
@@ -26,7 +26,9 @@
   "integration_id": "openstack",
   "assets": {
     "monitors": {},
-    "dashboards": {},
+    "dashboards": {
+      "OpenStack Controller Overview": "assets/dashboards/openstack-controller.json"
```

Comment:
We don't have an openstack_controller tile? Do you know why?

Reply:
openstack_controller is a re-work of openstack integration, so it's technically just openstack 2.0.